### ISSUE TITLE
After applying the reflow, python, C#, javascript, java,C++ code snippet is not visible: A11y_WCP URLs - ONNX Runtime_Home_Learn more about how to use ONNX Runtime with_Reflow 

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/onnxruntime#20118]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCP; #Win11;#FTP; #P3_WCP; #ONNX Runtime; #ChromiumEdge; #Reflow; #WCAG1.4.10; #A11yMAS;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".** 

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2

### Pre-Requisites:

1. Open the web page / screen of the web application that is to be tested.
2. Open Chrome/Edge dev Inspect tools. Keyboard shortcut "Ctrl+Shift+I".
3. Ensure devtools are customized to Dock location: "undock into separate window". Minimize devtools so they are not obscuring the page.
4. Zoom the browser window of the page to be tested to 200%.
5. Adjust the browser window's height and width to the required viewport of 320px*256px. As long as dev tools remain open while resizing the viewport size will be displayed in the upper right corner of the browser so the size can be verified.
6. Refresh the page.

**Alternatively:**

1. Change the OS display settings to a resolution of 1280x1024 at 100% dpi scaling.
2. Open the web page/ screen of the web application that is to be tested.
3. Zoom browser 400%.
4. Set the browser to full screen. Keyboard shortcut "F11" key.
5. If any browser sidebars are still visible (e.g., the Edge discover sidebar), these must be closed.
### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page is appeared.
3. Tab till "Learn more about how to use ONNX Runtime with" section.
4. Tab till "python, C#, JavaScript, java, C++" code snippet
5. Now check that after applying the reflow, python, C#, JavaScript, java, C++ code snippet is not visible.

### Actual Results:
After applying the reflow, python, C#, JavaScript, java, C++ code snippet is not visible.

### Expected Results:
After applying the reflow, python, C#, JavaScript, java, C++ code snippet should be visible without any overlap and truncations.

### User Impact:
Low visible user will face difficultly. There not able to see the python, C#, JavaScript, java, C++ code snippet. If code snippet is not visible in reflow.

### Attachment:

[python, C#, javascript, java,C++ code snippet is not visible.zip](https://github.com/microsoft/onnxruntime/files/14788926/python.C.javascript.java.C%2B%2B.code.snippet.is.not.visible.zip)

![codes is not visible](https://github.com/microsoft/onnxruntime/assets/142479541/728662e5-3a35-40c3-bc91-16efa2edaa1c)

**Comment by MaanavD** [comment]
Hey @PadminiGariki, this is intended behavior. The code snippets should not be visible on mobile.

**Comment by PadminiGariki** [comment]
Hi @MaanavD 
As per discuss with SME's, this issue should be fixed.
#Regressed:04-17-24;

**Comment by MaanavD** [comment]
Hi @PadminiGariki - sorry, can you clarify? I'm not sure what you mean by it 'should be fixed'? We don't intend to have users looking at those snippets on mobile. What is the requirement to have it as such?

**Comment by PadminiGariki** [comment]
Hi @MaanavD 
If that code snippets are accessible by web users. Then it should be accessible by mobile users also because low visible users are not able know that data. So, after applying the reflow, python, C#, JavaScript, java, C++ code snippet should be visible without any overlap and truncations.

#Regressed:04-18-24;

**Comment by PadminiGariki** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/training
Edge version:125.0.2535.92 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is fixed. Hence the bug is closed.
For reference attachment is added.

#Regressed:06-17-24; #Closed;

### Attachment:


[Recording-20240617_105458.webm](https://github.com/microsoft/onnxruntime/assets/142479541/79cf7711-e8ec-4244-bcb9-d5e514672e36)
